engine/src/main/java/com/datatorrent/stram/StreamingAppMasterService.java

```diff
@@ -644,6 +644,15 @@ protected void serviceStart() throws Exception
       if (SecurityUtils.isStramWebSecurityEnabled()) {
```

Comment:
I am not sure that we are on the same page. I did not suggest to move serviceStart() to SecurityUtils. The suggestion applies to the block of the code that starts with the if or try and deals with the security config. There are already methods like initAuth() in the SecurityUtils class that encapsulate knowledge of various Hadoop and apex security parameters. Spreading that knowledge/functionality across multiple classes should be avoided.